easybuild/tools/systemtools.py

```diff
@@ -406,7 +406,11 @@ def get_cpu_features():
                 # for Linux@POWER systems, no flags/features are listed, but we can check for Altivec
                 cpu_altivec_regex = re.compile("^cpu\s*:.*altivec supported", re.M)
                 if cpu_altivec_regex.search(proc_cpuinfo):
-                    cpu_feat = ['altivec']
+                    cpu_feat.append('altivec')
+                # VSX is supported since POWER7
+                cpu_power7_regex = re.compile("^cpu\s*:.*POWER(7|8|9)", re.M)
```

Comment:
What about extracting the trailing number and then compare >= 7?  Seems more future-proof (although even POWER9 isn't available yet).